llmfoundry/data/finetuning/dataloader.py

```diff
@@ -120,16 +120,22 @@ def build_finetuning_dataloader(cfg: DictConfig,
             tokenizer=tokenizer,
             local=cfg.dataset.local,
             remote=cfg.dataset.get('remote', None),
-            split=cfg.dataset.get('split'),
-            shuffle=cfg.dataset.get('shuffle', False),
-            predownload=cfg.dataset.get('predownload', 100_000),
-            keep_zip=cfg.dataset.get('keep_zip', False),
+            split=cfg.dataset.get('split', None),
```

Comment:
I think we require split to be specified in this code, so lets not change that right now. would need to look at the code more closely and decide.

Reply:
okay so i'll leave out the split arg then